app/routes/circulars/circulars.server.ts

```diff
@@ -272,6 +272,37 @@ export async function get(
   return result as Circular
 }
 
+/** Check if a Circular exists by ID. */
+export async function exists(
+  circularId: number,
+  version?: number
```

Comment:
Please remove this argument. The existence of a circular does not depend on version.